### PR TITLE
Updated API Functionality

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates :green_circle: 
- [ ] Implemented the "Refresh" button (receiving data from the API and parsing the JSON).
- [ ] Implemented the form "Submit" button (sending data to the API).
- [ ] Used arrow functions instead of the function keyword.
- [ ] Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/modules/async.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+    {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    },
+  );
```
Comment (codepantha) [inline src/modules/async.js:27]: - [ ] Please wrap this in a try and catch block. This is to enable you to catch any errors that might occur when using async await 👍

File: src/modules/async.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+};
+
+const get = async (gameId) => {
+  const score = await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+  );
+  return score.json().then((data) => data.result);
```
Comment (codepantha) [inline src/modules/async.js:34]: - [ ]  Please wrap this in a try and catch block. This is to enable you to catch any errors that might occur when using async await 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES

## Hi @lucabenaventew 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights ⭐ 
✔️ Descriptive PR
✔️ No linter errors
✔️ App working as expected

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @lucabenaventew, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**